modules/ppcp-uninstall/resources/js/ppcp-clear-db.js

```diff
@@ -0,0 +1,43 @@
+document.addEventListener(
+    'DOMContentLoaded',
+    () => {
+        const config = PayPalCommerceGatewayClearDb;
+        if (!typeof (config)) {
+            return;
+        }
+
+        const clearDbConfig = config.clearDb;
+
+        jQuery(document).on('click', clearDbConfig.button, function () {
```

Comment:
:thinking: 

You can do just `button.click...` or `button.addEventListener...`. Or `jQuery(buttonSelector).click...` / `document.querySelector...`

If it can be `null`, then either `if` + `return` or `button?.addEventListener...` (for jQuery `null` does not matter).

Reply:
done. 2bb8ea9246ea3cb0bf365fa10ae72e1b66bd5947